docs/contrib/Dev_Environment.md

```diff
@@ -0,0 +1,73 @@
+# Pre-Requisites
+
+## Access to a Large Language Model (LLM)
+
+- If you want to run Kai against a LLM you will likely need to configure a LLM API Key to access your service (unless running against a local model)
+  - We do provide a means of running Kai against previously cached data from a few models to aid demo flows. This allows you to run through the steps of using previously cached data without requiring access to a LLM. Note, if you do not provide LLM API access then the DEMO_MODE flow will only be able to replay previous cached responses.
+    - We call this 'DEMO_MODE', i.e. `DEMO_MODE=true make run-server`
+- Note that results vary widely between models.
+
+## Local Development
+
+Running Kai's backend involves running 2 processes:
```

Comment:
at least 2, the hub-importer is an optional 3rd

Reply:
+1 adding